ouroboros-network/src/Ouroboros/Network/PeerSelection/Churn.hs

```diff
@@ -347,10 +347,10 @@ peerChurnGovernor PeerChurnArgs {
     decreaseActiveBigLedgerPeers regime _ base targets =
       targets {
         targetNumberOfActiveBigLedgerPeers =
-          let praosSync = min 1 (targetNumberOfActiveBigLedgerPeers base)
+          let praosSyncTargets = min 1 (targetNumberOfActiveBigLedgerPeers base)
           in case regime of
-               ChurnBootstrapPraosSync -> praosSync
-               ChurnPraosSync -> praosSync
+               ChurnBootstrapPraosSync -> praosSyncTargets
+               ChurnPraosSync -> praosSyncTargets
```

Comment:
We use `praosSync` in other places. Maybe change it in other places as well?

Reply:
I think I renamed most of those to deadlineTargets as under Praos that's all we have and which is modified by stepping thru the churn loop. Here I decided to keep the praosSync prefix instead of renaming the let binding to deadlineTargets as maybe it would be slightly confusing.